etc/kayobe/kolla/config/prometheus/prometheus-alertmanager.yml

```diff
@@ -0,0 +1,32 @@
+{% raw %}
+global:
+  resolve_timeout: 5m
+  smtp_require_tls: true
+  slack_api_url: 'https://hooks.slack.com/services/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'
+route:
+  receiver: 'default'
+  group_by: [alertname, app]
+  group_interval: 10m
+  repeat_interval: 24h
+  routes:
+    - match:
+        severity: page
+      # Also generate Jira ticket
+      continue: true
+      receiver: page
+    - match:
+        severity: page
+      receiver: default
+receivers:
+  - name: 'default'
+    webhook_configs:
+    - url: 'http://localhost:9097/alert'
+      # Jiralert doesn't handle resolving issues, and complains if this is true
+      send_resolved: false
+  - name: 'page'
+    slack_configs:
+    - channel: '#iris-notifications'
+      title: '{{ '{{' }} .GroupLabels.app {{ '}}' }} detected  {{ '{{' }} .GroupLabels.alertname {{ '}}' }}'
+      text: '{{ '{{' }} range .Alerts {{ '}}' }}{{ '{{' }} .Annotations.description {{ '}}' }}\n{{ '{{' }} end {{ '}}' }} '
```

Comment:
can you not use raw tags or [unsafe](https://docs.ansible.com/ansible/latest/user_guide/playbooks_advanced_syntax.html#unsafe-or-raw-strings) for this?

Reply:
The problem with raw tags is that this gets templated twice, once by Kayobe, once by Kolla Ansible, and you can't nest raw tags (it is a go template). Arguably the double templating is unnecessary, but this works for now.